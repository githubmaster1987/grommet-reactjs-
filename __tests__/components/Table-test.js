// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Table from '../../src/js/components/Table';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('Table', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Table>
        <tbody>
          <tr>
            <td>
              first
            </td>
            <td>
              note 1
            </td>
          </tr>
          <tr>
            <td>
              second
            </td>
            <td>
              note 2
            </td>
          </tr>
          <tr>
            <td>
              third
            </td>
            <td>
              note 3
            </td>
          </tr>
        </tbody>
      </Table>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
