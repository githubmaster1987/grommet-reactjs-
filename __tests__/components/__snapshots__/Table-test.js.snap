exports[`Table has correct default options 1`] = `
<div
  className="grommetux-table">
  <table
    className="grommetux-table__table">
    <tbody>
      <tr>
        <td>
          first
        </td>
        <td>
          note 1
        </td>
      </tr>
      <tr>
        <td>
          second
        </td>
        <td>
          note 2
        </td>
      </tr>
      <tr>
        <td>
          third
        </td>
        <td>
          note 3
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
